src/main/webapp/js/script.js

```diff
@@ -319,15 +319,39 @@ function getStarRating(stars) {
 * Adds a featured map where the ID 'featured-map' is and the rest of the maps where the ID 'all-maps' is
 */
 function loadMaps() {
-  fetch('/load-mainpage-data').then(response => response.json()).then(async (data) => {
+  fetch('/load-mainpage-data?page='+0).then(response => response.json()).then(async (data) => {
     let featuredMap = createStaticMap(data[0].stageLocations, '400', data[0].gameID);
     let featuredMapCaption = createStaticMapCaption(data[0], 'featured-map-info');
     let featuredMapDiv = document.getElementById('featured-map');
     featuredMapDiv.classList.add('hoverable');
     featuredMapDiv.append(featuredMap);
     featuredMapDiv.append(featuredMapCaption);
+    if (data.length == 0 || data.length < 20) {
+      let moreMapsButton = document.getElementById('more-maps');
```

Comment:
I think it would look better to remove this button altogether in this case. You can set it as hidden.

Reply:
Okay sounds good!